tests/layer_section_by_section_tests.py

```diff
@@ -144,3 +144,13 @@ def test_process_order(self):
              "publication_date": "2010-10-10",
              "fr_volume": 22,
              "fr_page": 7676}])
+
+    def test_no_section_by_section(self):
+        notice = {
```

Comment:
This wasn't done previously in the code base, but it might be useful to add comments to tests to indicate why we expect it to do the thing we expect it to do. So, in this case something like: "There are sometimes notices without section by section analysis in them - let's ensure we handle those gracefully"